pkg/virt/domain_interface.go

```diff
@@ -120,5 +120,5 @@ type Domain interface {
 	// Update vcpu for a give domain
 	SetVcpus(uint) error
 	// Update current memory for a given domain
-	SetCurrentMemory(uint64) error
+	SetCurrentMemory(int64) error
```

Comment:
This name is better to be "ChangeMemory", "AdjustMemory" or something similar.  "SetCurrentMemory" gives the impression that the parameter is the new memory number, instead of the delta.

Reply:
fixed with AdjustDomainMemory(). thanks.